apps/examples/hello_tfmicro/main.cc

```diff
@@ -33,125 +33,136 @@ limitations under the License.
+
 #include <tinyara/init.h>
 
-/* Globals, used for compatibility with Arduino-style sketches. */
+
+// Globals, used for compatibility with Arduino-style sketches.
 namespace {
-tflite::ErrorReporter *error_reporter = nullptr;
-const tflite::Model *model = nullptr;
-tflite::MicroInterpreter *interpreter = nullptr;
-TfLiteTensor *input = nullptr;
-TfLiteTensor *output = nullptr;
+tflite::ErrorReporter* error_reporter = nullptr;
+const tflite::Model* model = nullptr;
+tflite::MicroInterpreter* interpreter = nullptr;
+TfLiteTensor* input = nullptr;
+TfLiteTensor* output = nullptr;
 int inference_count = 0;
 
-/* Create an area of memory to use for input, output, and intermediate arrays.
-   Finding the minimum value for your model may require some trial and error. */
+// Create an area of memory to use for input, output, and intermediate arrays.
+// Finding the minimum value for your model may require some trial and error.
 constexpr int kTensorArenaSize = 2 * 1024;
 uint8_t tensor_arena[kTensorArenaSize];
-} /* namespace */
-
-void HandleOutput(tflite::ErrorReporter *error_reporter, float x_value, float y_value)
-{
-	/* Log the current X and Y values */
-	error_reporter->Report("x_value: %f, y_value: %f\n", x_value, y_value);
-}
-
-/* The name of this function is important for Arduino compatibility. */
-void setup(void)
-{
-	/* Set up logging. Google style is to avoid globals or statics because of
-	 lifetime uncertainty, but since this has a trivial destructor it's okay.
-	 NOLINTNEXTLINE(runtime-global-variables) */
-	static tflite::MicroErrorReporter micro_error_reporter;
-	error_reporter = &micro_error_reporter;
-
-	/* Map the model into a usable data structure. This doesn't involve any
-	 copying or parsing, it's a very lightweight operation. */
-	model = tflite::GetModel(g_sine_model_data);
-	if (model->version() != TFLITE_SCHEMA_VERSION) {
-		error_reporter->Report(
-			"Model provided is schema version %d not equal "
-			"to supported version %d.",
-			model->version(), TFLITE_SCHEMA_VERSION);
-		return;
-	}
-
-	/* This pulls in all the operation implementations we need.
-	 NOLINTNEXTLINE(runtime-global-variables) */
-	static tflite::ops::micro::AllOpsResolver resolver;
-
-	/* Build an interpreter to run the model with. */
-	static tflite::MicroInterpreter static_interpreter(model, resolver, tensor_arena, kTensorArenaSize, error_reporter);
-	interpreter = &static_interpreter;
-
-	/* Allocate memory from the tensor_arena for the model's tensors. */
-	TfLiteStatus allocate_status = interpreter->AllocateTensors();
-	if (allocate_status != kTfLiteOk) {
-		error_reporter->Report("AllocateTensors() failed");
-		return;
-	}
-
-	/* Obtain pointers to the model's input and output tensors. */
-	input = interpreter->input(0);
-	output = interpreter->output(0);
-
-	/* Keep track of how many inferences we have performed. */
-	inference_count = 0;
+}  // namespace
+
+// The name of this function is important for Arduino compatibility.
+void setup() {
```

Comment:
You should follow TizenRT coding rules in the apps folder.

Reply:
This is sample application of TFMicro, I executed tools/cxxformatter.sh to make code guideline in sync with TizenRT